build.gradle

```diff
@@ -84,14 +84,13 @@ task ciBuild {
 
 
 wrapper {
-	gradleVersion = '4.10.3'
 	distributionType = Wrapper.DistributionType.ALL
 }
 
 
 buildScan {
-	licenseAgreementUrl = 'https://gradle.com/terms-of-service'
-	licenseAgree = 'yes'
+	termsOfServiceUrl = "https://gradle.com/terms-of-service"
```

Comment:
it's best to use the single quotes in this case, as it's a constant. the double quotes enable formatting / interpolation of variables.

Reply:
Ok; done